ansible_base/authentication/authenticator_plugins/local.py

```diff
@@ -52,6 +52,15 @@ def authenticate(self, request, username=None, password=None, **kwargs):
         if new_username != username:
             return None
 
+        auth_log_headers = (
+            f"HTTP_USER_AGENT: {request.META['HTTP_USER_AGENT'] if 'HTTP_USER_AGENT' in request.META else 'UNKNOWN'} "
+            f"HTTP_X_FORWARDED_FOR: {request.META['HTTP_X_FORWARDED_FOR'] if 'HTTP_X_FORWARDED_FOR' in request.META else 'UNKNOWN'} "
+            f"REMOTE_ADDR: {request.META['REMOTE_ADDR'] if 'REMOTE_ADDR' in request.META else 'UNKNOWN'} "
+            f"REMOTE_HOST: {request.META['REMOTE_HOST'] if 'REMOTE_HOST' in request.META else 'UNKNOWN'}"
+        )
+
+        logger.info(f"Login attempt for user: {username} {auth_log_headers}")
```

Comment:
Should we make this a debug log level or do we feel like these headers are info level worthy?